.github/workflows/stale.yaml

```diff
@@ -0,0 +1,31 @@
+name: Close Stale
```

Comment:
I'm a little dubious about this one so early in the project's life.  It's not as though we are besieged with input, also I'm retired and Quamina is if not my primary hobby it's close to the top, and if we don't like a PR or an issue we should boot it or close it.

Reply:
Sure, happy to remove this one for now!